xio-core/src/main/java/com/xjeffrose/xio/config/ConfigReloader.java

```diff
@@ -99,8 +100,16 @@ public T init(String file) {
     return metadata.value;
   }
 
-  public void start() {
+  @VisibleForTesting
+  public void setUpdater(BiConsumer<T, T> updater) {
```

Comment:
maybe make this package private or protected

Reply:
done